README.rst

```diff
@@ -18,3 +18,16 @@ Some subtitle goes here
         #. just to test
     #. some kinds
 #. of stuffs.
+
+This is a new section
+=====================
+
+.. A simple grid table.
+
++-------+-------+
```

Comment:
I think that this table has something wrong... Check out the reStructuredText documentation before merging.